src/libstd/sys/windows/process.rs

```diff
@@ -35,12 +35,6 @@ use sys_common::{AsInner, FromInner};
 // Command
 ////////////////////////////////////////////////////////////////////////////////
 
-fn mk_key(s: &OsStr) -> OsString {
-    FromInner::from_inner(sys::os_str::Buf {
-        inner: s.as_inner().inner.to_ascii_uppercase()
```

Comment:
Can the existing behavior be preserved? We can discuss the specifics in a further issue if you'd like to make a case for removal.

AFAIK Windows has case insensitive keys, so we canonicalize to uppercase.